composer/datasets/in_context_learning_evaluation.py

```diff
@@ -306,7 +306,8 @@ def collate_fn(self, data):
             'generation_length': self.max_answer_length,
             'generation_kwargs': {
                 'pad_token_id': self.pad_tok_id,
-                'use_cache': True
+                'use_cache': True,
+                'eos_token_id': self.tokenizer.eos_token_id
```

Comment:
assuming this is truly required, we should check somewhere that `eos_token_id` is not `None`, and error if it is

Reply:
I don't think it shouldn't necessarily error because it's possible some models could have been trained to use a different token to indicate ends of sequences and we wouldn't want to completely rule them out, right?

Reply:
how would they specify that other than via eos token?

Reply:
not sure, I was just assuming it's possible since couldn't you theoretically choose to put whatever token you want at the ends of your sequences? I could be wrong